pom.xml

```diff
@@ -9,21 +9,20 @@
         <version>3.1</version>
     </parent>
     <artifactId>sirius-web</artifactId>
-    <version>4.5</version>
+    <version>4.5.1</version>
     <name>SIRIUS web</name>
     <description>Provides a modern and scalable web server as SIRIUS module</description>
 
     <dependencies>
-
         <dependency>
             <groupId>com.scireum</groupId>
             <artifactId>sirius-kernel</artifactId>
-            <version>3.0.1</version>
+            <version>3.1</version>
         </dependency>
         <dependency>
             <groupId>com.scireum</groupId>
             <artifactId>sirius-kernel</artifactId>
-            <version>[3.0.1,)</version>
+            <version>[3.1,)</version>
```

Comment:
Please use a specific version here as this can cause tests to fail after a newer version of sirius-kernel was released